core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -603,6 +602,21 @@ class AutoTuner(
     recommendClassPathEntries()
   }
 
+  def getShuffleManagerClassName() : String = {
+    val shuffleManagerVersion = appInfoProvider.getSparkVersion.get.filterNot("().".toSet)
+    val finalShuffleVersion : String = if (platform.getName == PlatformNames.DATABRICKS_AWS
+      || platform.getName == PlatformNames.DATABRICKS_AZURE) {
+      val dbVersion = appInfoProvider.getProperty(
+        "spark.databricks.clusterUsageTags.sparkVersion").getOrElse("")
+      dbVersion match {
```

Comment:
For the `if-condition`, I would think that we should not rely on the platform input.
Instead we look for a property that distinguished eventlogs generated on Databricks.
For example,
- if this configuration `spark.databricks.clusterUsageTags.sparkVersion` is always set for all Databricks executions, then all we need to do is to check that this config entry is not emptyt
- If the above is not always true, then we can use pattern match of `SparkVersion` pulled from the app configuration indicates it is a databricks-spark

Deciding the platform from the configuration is going to be a good step toward improving the analysis. We are going to need that anyway.

Reply:
Yeah, I sort of alluded to that when I first put the PR up. I like trying out option#1 above. Will update.

Reply:
Addressed